source/index.js

```diff
@@ -165,11 +169,21 @@ module.exports = async (input = 'patch', options) => {
 		{
 			title: 'Bumping version using Yarn',
 			enabled: () => options.yarn === true,
+			skip: () => {
+				if (options.preview) {
+					return 'Command not executed in preview mode: yarn version…';
```

Comment:
```suggestion
					return '[Preview] Command not executed: yarn version…';
```

What do you think about this style? The current one is a bit verbose. I think we should also give the command a different color, maybe cyan.